services/automated-actions.js

```diff
@@ -0,0 +1,15 @@
+import schedule from 'node-schedule'
+import firebase from 'firebase'
+import moment from 'moment'
+
+let checkDatabase = schedule.scheduleJob('* * */1 * *', () => {
+  firebase.database().ref('/games').once('value').then(function(snapshot) {
+    for (let key in snapshot.val()) {
```

Comment:
Use `.forEach()` to loop through Arrays. [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)

Reply:
I thought we discussed the forEach possibility and decided this was better (?) . I think we wanted snapshot.val() or something and we thought that this would be better

Reply:
Oh yeah, we decided not to user `forEach` because `snapshot.val()` is a dictionary, which means `forEach` doesn't work on the data type.